### DISCUSSION TITLE
Two new maintainers

### DISCUSSION THREAD

**Discussion by nikomatsakis** [lalrpop/lalrpop#832] [General]
I wanted to announce that @Pat-Lafon and @dburgener have been added as maintainers to the LALRPOP respository. Thanks for all your contributions!

The list of maintainers can be found here (I believe membership is publicly visible): 

https://github.com/orgs/lalrpop/teams/maintainers

**Comment by yannham** [comment]
Great news! @Pat-Lafon and @dburgener have been fairly active and helpful these days, so I warmly welcome them
